demisto_sdk/commands/validate/initializer.py

```diff
@@ -504,15 +505,20 @@ def git_paths_to_basecontent_set(
         basecontent_with_path_set: Set[BaseContent] = set()
         invalid_content_items: Set[Path] = set()
         non_content_items: Set[Path] = set()
+        git_util = GitUtil.from_content_path()
+        current_git_sha = git_util.get_current_commit_hash()
```

Comment:
Not sure I fully understand all the change here, but are we sure it won't affect the way we load changed files?

Reply:
1. renaming `git_sha` arg to `prev_ver` just to avoid confusion between prev and current versions
2. passed `current_git_sha` to the object's `git_sha`. from what I saw, the git_sha attribute is used only in:
    a) related_files.py (which is what I fixed)
    b) integration and script parsers to get the code - if git_sha not passed, it uses the current working dir (equivalent to current git sha), so allegedly nothing was changed.
3. passed `prev_ver` to old object's `git_sha`.

With 2+3, we can calculate the git_status of related files correctly for each object (see changes in related_files.py and git_utils.py)